swarm/api/routes/wisdom.py

```
@@ -505,6 +519,18 @@ async def get_wisdom_content(
     """
+    try:
+        validate_path_component(artifact_name, "artifact_name")
+    except ValueError as e:
+        raise HTTPException(
+            status_code=400,
+            detail={
+                "error": "invalid_artifact_name",
+                "message": str(e),
+                "details": {"artifact_name": artifact_name},
+            },
+        )
```

Ruff (0.15.5)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)